api/webhook-vs-polling.md

```diff
@@ -64,25 +64,24 @@ Use `GET` when:
 {% hint style="info" %}
 Pro's:
 
-* Semi real time, receive the order within seconds.
-* You can filter on which order events to receive \(in the company integration settings only\).
+* Semi real time, receive the order or shipment within seconds.
+* You can filter on which order or shipment events to receive, in the order & shipment webhook settings in your company profile.
 * You do not have to build or configure the polling pattern.
 
 Con's:
 
-* You need to fetch the order.
-* Basic authentication is not yet supported for the GET `/order/:orderId` API. Send a support request if you need it.
-* You cannot see what order event happened.
+* You need to fetch the order or shipment.
+* You cannot see what order or shipment event happened.
 * You need to build or configure a webhook at your side.
 * You need to publish the webhook on the internet \(web server and firewall required\).
 * You need to obtain and configure a public SSL certificate.
 {% endhint %}
 
 ## The polling pattern
 
-Check if there are new or updated orders every polling period, typically 5 minutes, by using the last updated date time stamp of the last fetched order.
+Check if there are new or updated orders or shipments every polling period, typically 5 minutes, by using the last updated date time stamp of the last fetched order or shipment.
 
-The polling pattern is most suitable for companies with low volume order responses and not willing to invest in a web server, firewall and SSL certificate.
+The polling pattern is most suitable for companies with low volume orders, and not willing to invest in a web server, firewall and SSL certificate.
```

Comment:
Nit: somehow I'm reading this quite negative "not willing to invest in..."

Reply:
That is what I mean, Hollandse Zuinigheid